### DISCUSSION TITLE
⛔ Add Instagram to the Share Buttons widget

### DISCUSSION THREAD

**Discussion by BASCweb** [elementor/elementor#18125] [Feature Request]
Hello,

Just wondering if its possible to have Instagram added as an option to the share buttons element in the editor

Thanks

**Comment by LoRosset** [comment]
Hi there,

I would really like Instagram to be part of the widget "share buttons" please ! That would be awesome.

Best regards

**Reply by loucasl** [reply]
You can't share links to Instagram, so this isn't possible

**Comment by nicholaszein** [comment]
Hello.

Instagram doesn't have a share feature, and you can't share content via Instagram.

Kind regards
